upgrade/satellite.py

```diff
@@ -270,6 +268,5 @@ def satellite6_zstream_upgrade():
     reboot(180)
     host_ssh_availability_check(env.get('satellite_host'))
     # Test the Upgrade is successful
-    set_hammer_config()
-    hammer('ping')
+    run('hammer ping', warn_only=True)
```

Comment:
@jyejare shall we remove it here as well?

Reply:
Its ok, even if we keep this, It is not going to have any impact.

Reply:
@ntkathole , Removed in all PRs.